proto/momento/auth.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "momento.auth";
+
+package momento.auth;
+
+service Auth {
+  rpc Login (_LoginRequest) returns (stream _LoginResponse) {}
```

Comment:
This isn't entirely related to this PR, but the way this is trending it seems as though this will be a cellular login endpoint.  Have we had any discussion about how this would relate to / interact with any global state that we might want to store about auth?  Or are we planning for all of our auth state to be entirely cellular?

Reply:
will slack you